src/display/tests/test_fix_swapped_route_reverify.py

```
@@ -0,0 +1,85 @@
+            self.editable_route.route["features"][0]["geometry"]["coordinates"],
+            [[11.2, 60.2], [11.3, 60.3]],
+        )
+
+    def test_force_overrides_the_reverification(self):
+        out = StringIO()
+        call_command(
+            "fix_swapped_route",
+            str(self.editable_route.id),
+            "--force",
+            "--backup-dir",
+            "/tmp/test_fix_swapped_route_reverify_backups",
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_fix_swapped_route_reverify_backups"

(S108)